course_1_1/Program.cs

```diff
@@ -10,19 +10,86 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Please enter a radius");
-            string rstr = Console.ReadLine();
-            int.TryParse(rstr, out int r);
-            Console.WriteLine($"The square of the round {Program.PerformCalculation(r)}");
-            
-        }
+            ////Задание 1
+
+            //int xl;
```

Comment:
It would be better and more readable if you put all tasks in separate methods and commented invocations only.